sqlx-cli/src/opt.rs

```diff
@@ -111,13 +111,21 @@ pub enum MigrateCommand {
         /// List all the migrations to be run without applying
         #[clap(long)]
         dry_run: bool,
+
+        /// Ignore applied migrations that missing in the resolved migrations
```

Comment:
The English is a bit broken here. I believe this is meant to say "Ignore applied migrations that _are_ missing in the resolved migrations." However, even with that I'm still confused, as I don't think the docs ever define what a "resolved migration" is.

I could also imagine "--ignore-missing" being ambiguous. When used in the context of running migrations found in a source, one possible way to read this would be that we're ignoring migrations in the source that are missing from the database.

Perhaps "--no-check-missing"? Then it makes it more clear that what we're doing isn't ignoring any migration that might possibly be run, but instead disabling some check.

Another way of doing this would be to make the default behavior be to not perform such a check, and then make the behavior to do the check opt-in with `--check-missing`. The docs could read something like "Check before running migrations that the migration source contains all previously-applied migrations". Although this would be confusing if the behavior of checking for changed migrations would still be default.

I guess what I'm trying to say here is that I like the idea of no longer forcing this check, I'm just wondering if there's a better way of naming / handling how to opt in/out of it.

Reply:
I definitely want to keep the check-by-default. I'm open to changing the name of the option, perhaps in the next breaking release. If you feel strongly about a particular option, please open an issue. Personally, I think `--ignore-missing` is fine and we can do a pass on the docs later to help explain what a "missing" migration is.